.github/actions/build-repository/action.yml

```diff
@@ -59,6 +62,9 @@ runs:
     - name: Install
       shell: bash
       run: npm install
+    - name: Revert package(-lock).json changes
```

Comment:
- Is it also necessary to revert `package.json` or would we be good with just reverting `package-lock.json`?
- Do we need all this logic or could we do just something like `git restore package*.json`?
- Nitpick: double space in `true)  |`

Reply:
We need to revert both: they both end up with local paths to dependencies due to the "Path local dependencies" step. The logic is needed because the files are sometimes in subfolders.